docs/source/userguide/beginners_guide.rst

```diff
@@ -12,6 +12,12 @@ New users should familiarise themselves with `Conda <https://anaconda.org/>`_.
 
 FESTIM is under version control with `git <https://git-scm.com/>`_. Even though users don't need git to install FESTIM, it is convenient to understand how it works. You can find `git tutorials <https://git-scm.com/doc>`_ to help you getting started. The `FESTIM source code <https://github.com/festim-dev/FESTIM>`_ is hosted on GitHub. Signing up for free to GitHub is recommended to receive the latest updates, follow the development and submit `bug reports <https://github.com/festim-dev/FESTIM/issues/new/choose>`_.
 
+FESTIM can be run in parallel on N cores using the command: 
+
+.. testcode::
```

Comment:
```suggestion
.. code::
```

If you use ``testcode`` this code block will be tested as if it was a python code block. This obviously won't work and this is why the documentation test is failing. 
Simply replace this by a normal code block.